simpleLooping/incrementBinaryList.py

```diff
@@ -18,3 +18,24 @@
 #   incrementBinaryList([1, 0, 0, 1]) # Return [0, 1, 0, 1]
 #   incrementBinaryList([1, 1, 0, 1]) # Return [0, 0, 1, 1]
 #   incrementBinaryList([1, 1, 1, 1]) # Return [0, 0, 0, 0, 1]
+from typing import List
+
+def incrementBinaryList(lst: List[int]) -> List[int]:
+    lenLst = len(lst)
+    i = 0
+    listBinary = []
+    carry = 1
+
+    while i < len(lst):
+        if lst[i] + carry == 2:
+            listBinary.append(0)
+            carry = 1
+        else:
+            listBinary.append(lst[i] + carry)
+            carry = 0
+        i += 1
+    
+    if carry == 1:
```

Comment:
Just use "if carry:"

The variable "carry" can only be one of the two values 0 and 1, so if it is 1 then it will be True. Don't really need to compare it with 1 and does not add meaning to it.

Reply:
fixxed.